registry/tests/versioning.unit.spec.ts

```diff
@@ -14,15 +14,19 @@ const testUser = Object.freeze({
 describe('Versioning Unit', () => {
     let db: VersionedKnex;
     let versionSevice: Versioning;
-    before(async () => {
+    before(async function() {
+        this.timeout(10 * 1000);
```

Comment:
which changes caused setting this timeout?

Reply:
Once I had build failed because of the timeout error.